src/main/java/EduardOOPTask/Notebook.java

```diff
@@ -0,0 +1,20 @@
+package EduardOOPTask;
+
+public class Notebook extends Computer {
+
+    public Notebook(String computerType, String typeOfProcessor, int ramGb, double screenSize) {
+        super(computerType, typeOfProcessor, ramGb, screenSize);
+    }
+
+    public Notebook() {
+    }
+
+    @Override
+    public void setRamGb(int ramGb) {
```

Comment:
Actually this class has no use, because you may directly - using a Computer class - to change the ram.

Reply:
Done